docs/commands/Should.mdx

```diff
@@ -648,7 +655,8 @@ Accept wildcard characters: False
 
 ### -ExceptionType
 
-{{ Fill ExceptionType Description }}
+Depends on operator being used.
+See `Get-ShouldOperator -Name \<Operator\>` for help.
```

Comment:
```suggestion
See `Get-ShouldOperator -Name <Operator>` for help.
```

@bravo-kernel Bug? Should it escape inside inline code? I can remove the `< >` in code if it's bad practice - didn't really think too much about any consequences.

Reply:
I am not sure this is inline code. It looks like some PlatyPS escaped output that we have not encounterd before so I think I could improve the parser and simply remove the escaping. This way you can keep using `<Operator>` in the source which is totally valid IMO. Do you agree?

Reply:
I meant inline as in I used the backticks directly in the comment-help to format it correctly for the website. You're right that it's likely platyPS that escapes it, but I know you modify some of these code-parts so wanted to check 🙂

Reply:
Aha, I see now. I will investigate this one. Thanks for clarifying.